modules/core/plugin-container/src/main/java/org/rhq/core/pc/inventory/InventoryManager.java

```diff
@@ -1945,7 +1945,8 @@ private Configuration mergeExistingResourcePluginConfiguration(Resource resource
             Property discoveredProp = pluginConfig.get(propertyName);
             // Only overrides the property if it is a read-only property
             // See BZ: 1379834
-            if (propDefs.get(propertyName).isReadOnly()) {
+            PropertyDefinition propDefinition = propDefs.get(propertyName);
```

Comment:
Maybe a bit of paranoia, but, is `propDefs` always non `null` ?
https://github.com/rhq-project/rhq/pull/273/files#diff-32f6b984f4c7846e751c48497839e07aR1938